JSTests/wasm/stress/exnref-table.js

```diff
@@ -0,0 +1,4 @@
+var data = [ 0x0, 0x61, 0x73, 0x6d, 0x1, 0x0, 0x0, 0x0, 0x1, 0x99, 0x0, 0x3, 0x60, 0x3, 0x7f, 0x7f, 0x7f, 0x1, 0x7f, 0x60, 0xb, 0x7e, 0x7f, 0x7f, 0x7f, 0x7c, 0x7f, 0x7f, 0x7f, 0x7f, 0x7f, 0x7f, 0x0, 0x60, 0x0, 0x0, 0x4, 0x8d, 0x0, 0x3, 0x69, 0x1, 0x1, 0x35, 0x6a, 0x1, 0x17, 0x1c, 0x6b, 0x1, 0x15, 0x41 ];
```

Comment:
It seems that the `data` corresponds to the following WAT:

```wat
(module
  (type (;0;) (func (param i32 i32 i32) (result i32)))
  (type (;1;) (func (param i64 i32 i32 i32 f64 i32 i32 i32 i32 i32 i32)))
  (type (;2;) (func))
  (table (;0;) 1 53 exnref)
  (table (;1;) 23 28 arrayref)
  (table (;2;) 21 65 structref)
)
```

However, for this PR's test, I believe the `func` definitions and the `arrayref` and `structref` tables are unnecessary.

Shouldn't the WAT be shortened to something like the following?

```js
// (module
//   (table (;0;) 1 53 exnref)
// )
var data = [0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00, 0x04, 0x05, 0x01, 0x69, 0x01, 0x01, 0x35];
```

At least, the corresponding WAT should be included as a comment for better readability.

Reply:
This is coming from fuzzer bug and not particularly focusing on reducing the test case, so I'll leave it as is.

Reply:
> This is coming from fuzzer bug and not particularly focusing on reducing the test case since it is already small, so I'll leave it as is.

Can we at least include the current WAT as a comment? It's pretty annoying to reconstruct it from the binary when you need it.

Reply:
Done.